packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -193,6 +193,11 @@ class UserMessages {
   String get visualStudioMissing =>
       'Visual Studio not installed; this is necessary for Windows development.\n'
       'Download at https://visualstudio.microsoft.com/downloads/.';
+  String get visualStudioIsPrerelease => 'The current Visual Studio installation is a pre-release version';
```

Comment:
We should probably provide a bit more context about what effect this may have. Let's add another sentence: "It may not be supported by Flutter yet."

Reply:
Done